agent/agent_test.go

```diff
@@ -1859,7 +1859,7 @@ func TestAgent_PersistCheck(t *testing.T) {
 		t.Fatalf("err: %s", err)
 	}
 	if !bytes.Equal(expected, content) {
-		t.Fatalf("bad: %s", string(content))
+		t.Fatalf("bad: %s != %s", string(content), expected)
```

Comment:
Totally fine like this but I tend to rewrite old assertion messages like that to use `require.Equal` which has much nicer output as well as being fewer lines to read.